Chess/src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
+{
+	COORD topLeft = { 0, 0 };
+	HANDLE console = GetStdHandle(STD_OUTPUT_HANDLE);
+	CONSOLE_SCREEN_BUFFER_INFO screen;
+	DWORD written;
+
+	GetConsoleScreenBufferInfo(console, &screen);
+	FillConsoleOutputCharacterA(
+		console, ' ', screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	FillConsoleOutputAttribute(
+		console, FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_BLUE,
+		screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	SetConsoleCursorPosition(console, topLeft);
+}
+// create the GUI - ASCII art
+void Chess::setFrames() 
+{ 
+	// set all to ' ' (space bar ascii value 32) instead of 0  
+	{
+		for (size_t row = 0; row < _SIZE; ++row)
+			for (size_t col = 0; col < _SIZE; ++col)
```

Comment:
You can simplify the nested loops for setting the inside frame by using a single loop for both row and column indices. This will eliminate the need for separate loops for rows and columns and reduce code duplication.